src/System.IO.Pipelines.Extensions/PipeOutput.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Buffers;
+
+namespace System.IO.Pipelines
+{
+    public struct PipeOutput : IOutput
```

Comment:
Your API listing at the top of this PR does not show PipeOutput and PipeSequence

Reply:
Listing is for main pipelines assembly, this type is in extensions.